components/org.wso2.carbon.identity.application.authenticator.fido2/src/main/java/org/wso2/carbon/identity/application/authenticator/fido2/core/WebAuthnService.java

```diff
@@ -94,18 +96,22 @@
     private static ArrayList origins = null;
 
     public Either<String, RegistrationRequest> startRegistration(@NonNull String origin)
-            throws JsonProcessingException, FIDO2AuthenticatorException {
+            throws JsonProcessingException, FIDO2AuthenticatorClientException {
 
         readTrustedOrigins();
         if (!origins.contains(origin.trim())) {
-            throw new FIDO2AuthenticatorException(FIDO2AuthenticatorConstants.INVALID_ORIGIN_MESSAGE);
+            throw new FIDO2AuthenticatorClientException(FIDO2AuthenticatorConstants.INVALID_ORIGIN_MESSAGE,
+                    FIDO2AuthenticatorConstants.ClientExceptionErrorCodes.ERROR_CODE_START_REGISTRATION_INVALID_ORIGIN
+                            .getErrorCode());
         }
 
         URL originUrl;
         try {
             originUrl = new URL(origin);
         } catch (MalformedURLException e) {
-            throw new FIDO2AuthenticatorException(FIDO2AuthenticatorConstants.INVALID_ORIGIN_MESSAGE);
+            throw new FIDO2AuthenticatorClientException(FIDO2AuthenticatorConstants.INVALID_ORIGIN_MESSAGE,
```

Comment:
include 'e' also. Check and fix other similar usages

Reply:
Fixed in https://github.com/wso2-extensions/identity-local-auth-fido/pull/41/files#diff-f01c1476dba8be79bcd0e8f51ffa9a94R114